src/analysis/special_functions/pow_tactic.lean

```diff
@@ -0,0 +1,209 @@
+/-
```

Comment:
Hmm, sorry but I think it makes no sense to have the extension separate from the lemmas that feed it.

Reply:
... but then all the lemmas that feed it need to be in the same place, and we end up back at the same massive single-file hairball we had to start with.